client/plasma_cash/interface.go

```diff
@@ -40,7 +40,7 @@ type ChainServiceClient interface {
 	SubmitBlock() error
 	Deposit(deposit *pctypes.DepositRequest) error
 
-	SendTransaction(slot uint64, prevBlock *big.Int, denomination *big.Int, newOwner string, prevOwner string, sig []byte, hash []byte) error
+	SendTransaction(slot uint64, prevBlock *big.Int, denomination *big.Int, newOwner string, prevOwner string, sig []byte, hash []byte, replayProtection *pctypes.ReplayProtection) error
```

Comment:
The `replayProtection` param just needs to be a `[]byte` signature from signing the hash containing the sender, nonce & tx hash.